### PR TITLE
[VEN-2496]: Deployments of oracles on ArbitrumOne

### PR DESCRIPTION
**Pull request by Debugger022**
## Description

Resolves #VEN-2496

This PR adds deployments of oracles on ArbitrumOne.

Deployed addresses:
```
BoundValidator: 0x2245FA2420925Cd3C2D889Ddc5bA1aefEF0E14CF
DefaultProxyAdmin: 0xF6fF3e9459227f0cDE8B102b90bE25960317b216
ResilientOracle: 0xd55A98150e0F9f5e3F6280FC25617A5C93d96007
SequencerChainlinkOracle: 0x9cd9Fcc7E3dEDA360de7c080590AaD377ac9F113
RedStoneOracle: 0xF792C4D3BdeF534D6d1dcC305056D00C95453dD6
```
```
Arbitrum Sepolia:
Redstone Oracle: 0x15058891ca0c71Bd724b873c41596A682420613C
```
## Checklist

- [ ] I have updated the documentation to account for the changes in the code.
- [ ] If I added new functionality, I added tests covering it.
- [ ] If I fixed a bug, I added a test preventing this bug from silently reappearing again.
- [ ] My contribution follows [Venus contribution guidelines](docs/CONTRIBUTING.md).

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -38,7 +38,7 @@
     "@nomicfoundation/hardhat-network-helpers": "^1.0.8",
     "@openzeppelin/contracts": "^4.6.0",
     "@openzeppelin/contracts-upgradeable": "^4.7.3",
-    "@venusprotocol/governance-contracts": "^2.0.0",
+    "@venusprotocol/governance-contracts": "^2.1.0-dev.1",
```
Comment (chechu) [inline package.json:41]: Let's avoid `dev` versions, if it's possible

Comment (Debugger022) [reply package.json]: Done https://github.com/VenusProtocol/oracle/pull/192/commits/5abef9d7e358d651a55555d2e1b28c28a3c81160

File: helpers/deploymentConfig.ts [hunk 2]
```diff
@@ -863,6 +871,38 @@ export const assets: Assets = {
       oracle: "chainlinkFixed",
     },
   ],
+  arbitrumone: [
+    {
+      token: "WBTC",
+      address: "0x2f2a2543b76a4166549f7aab2e75bef0aefc5b0f",
+      price: "63000000000000000000000",
```
Comment (chechu) [inline helpers/deploymentConfig.ts:878]: I would not add the `price` entries into this items, to avoid confusion

**Review by chechu — APPROVED** [review]

File: helpers/deploymentConfig.ts [hunk 3]
```diff
@@ -240,6 +241,13 @@ export const chainlinkFeed: Config = {
     ARB: "0xD1092a65338d049DB68D7Be6bD89d17a0929945e",
     WETH: "0xd30e2101a97dcbAeBCBC04F14C3f624E67A35165",
   },
+  arbitrumone: {
```
Comment (chechu) [inline helpers/deploymentConfig.ts:244]: Should we also updated `redstoneFeed`, adding the XVS addresses for arbitrum one (and Ethereum)?

File: helpers/deploymentConfig.ts (hunk 2 above)
Comment (Debugger022) [reply helpers/deploymentConfig.ts:878]: Should i remove prices from old listed ones too?

File: helpers/deploymentConfig.ts [hunk 4]
```diff
@@ -240,6 +241,13 @@ export const chainlinkFeed: Config = {
...
+    WBTC: "0xd0C7101eACbB49F3deCcCc166d238410D6D46d57",
```
[the remark this answers is not in the document]
Comment (Debugger022) [reply helpers/deploymentConfig.ts]: Done https://github.com/VenusProtocol/oracle/pull/192/commits/7468ae144931475c087eccc662ca486f2675f865

File: helpers/deploymentConfig.ts (hunk 3 above)
Comment (Debugger022) [reply helpers/deploymentConfig.ts:243]: Done https://github.com/VenusProtocol/oracle/pull/192/commits/8c1b942ada08033c44c22fe20ab6eeb7062cf09f

File: deploy/2-deploy-redstone-oracle.ts [hunk 5]
```diff
@@ -4,28 +4,48 @@ import { HardhatRuntimeEnvironment } from "hardhat/types";
 
 import { ADDRESSES } from "../helpers/deploymentConfig";
 
+const ARBITRUM_SEQUENCER = "0xFdB631F5EE196F0ed6FAa767959853A9F217697D";
```
[the remark this answers is not in the document]
Comment (Debugger022) [reply deploy/2-deploy-redstone-oracle.ts]: Done https://github.com/VenusProtocol/oracle/pull/192/commits/3cd18f5b3be060eb9fcff1ec23a3f29203528d72

File: deploy/1-deploy-oracles.ts [hunk 6]
```diff
@@ -161,7 +161,13 @@ const func: DeployFunction = async function ({ getNamedAccounts, deployments, ne
   }
 
   const resilientOracle = await hre.ethers.getContract("ResilientOracle");
-  const chainlinkOracle = await hre.ethers.getContract("ChainlinkOracle");
+
+  let chainlinkOracle;
+  if (networkName === "arbitrumone") {
```
[the remark this answers is not in the document]
Comment (Debugger022) [reply deploy/1-deploy-oracles.ts]: Done https://github.com/VenusProtocol/oracle/pull/192/commits/3cd18f5b3be060eb9fcff1ec23a3f29203528d72

File: deploy/2-deploy-redstone-oracle.ts [hunk 7]
```diff
@@ -4,28 +4,48 @@ import { HardhatRuntimeEnvironment } from "hardhat/types";
...
-    log: true,
-    deterministicDeployment: false,
-    args: [],
-    proxy: {
-      owner: proxyOwnerAddress,
-      proxyContract: "OptimizedTransparentProxy",
-      execute: {
-        methodName: "initialize",
-        args: network.live ? [ADDRESSES[network.name].acm] : [],
+  if (network.name === "arbitrumone") {
```
[the remark this answers is not in the document]
Comment (Debugger022) [reply deploy/2-deploy-redstone-oracle.ts]: Done https://github.com/VenusProtocol/oracle/pull/192/commits/3cd18f5b3be060eb9fcff1ec23a3f29203528d72

File: helpers/deploymentConfig.ts (hunk 2 above)
Comment (Debugger022) [reply helpers/deploymentConfig.ts]: Removed it from the places where it was not needed.
https://github.com/VenusProtocol/oracle/pull/192/commits/3cd18f5b3be060eb9fcff1ec23a3f29203528d72

File: deploy/2-deploy-redstone-oracle.ts [hunk 8]
```diff
@@ -37,6 +57,6 @@ const func: DeployFunction = async function ({ getNamedAccounts, deployments, ne
 };
 
 func.skip = async ({ network }: HardhatRuntimeEnvironment) =>
-  !["hardhat", "bscmainnet", "bsctestnet", "sepolia", "ethereum"].includes(network.name);
+  !["hardhat", "bscmainnet", "bsctestnet", "sepolia", "ethereum", "arbitrumone"].includes(network.name);
```
[the remark this answers is not in the document]
Comment (Debugger022) [reply deploy/2-deploy-redstone-oracle.ts:44]: @chechu, Will we be setting hardcoded price in this **Redstone** oracle for **ArbitrumSepolia**?

[the remark this answers is not in the document]
Comment (chechu) [reply deploy/2-deploy-redstone-oracle.ts:44]: Yes, we can/should set a direct price for XVS on Arbitrum sepolia

[the remark this answers is not in the document]
Comment (Debugger022) [reply deploy/2-deploy-redstone-oracle.ts]: Done https://github.com/VenusProtocol/oracle/pull/192/commits/1373712404b84888ab93ca9adf51965a6f64ca0b

**Review by chechu — APPROVED** [review]

**Pull request merged**